### PR TITLE
test-network: redis usage with storj sim - default and customizable options

### PR DESCRIPTION
**Pull request by jenlij**

### CODE REVIEW

File: Test-network.md [hunk 1]
```diff
@@ -140,23 +140,57 @@ $ storj-sim network run
 ### Using Redis
 A few different processes can use redis as their database. Live accounting cache and revocation db both need to use redis on the Satellite. They will exist as separate databases on the same redis instance. For example, live accouning may use `redis://127.0.0.1:6379?db=0` while revocation may use `redis://127.0.0.1:6379?db=1`. 
 
-The default on storj-sim is to create a test redis server automatically and assign databases to processes incrementally based on how many Satellites are run. For example, if you're running 2 satellites, a configuration like the following may be generated.
+By default, storj-sim will start a redis server for each Satellite that is run and write the respective connection string for each database to a redis.conf file. You must have `redis-server` installed locally for this process to run. You can download redis at https://redis.io/download.
```
Comment (VinozzZ) [inline Test-network.md]: Maybe we can add that for mac, you can install it with `brew install redis`

File: Test-network.md [hunk 2]
```diff
@@ -140,23 +140,57 @@ $ storj-sim network run
...
+// satellite 0
+--live-accounting.db: redis://127.0.0.1:6379?db=0
+--server.revocation-dburl: redis://127.0.0.1:6379?db=1
+```
+
+```
+// satellite 1
+--live-accounting.db: redis://127.0.0.1:6379?db=2
+--server.revocation-dburl: redis://127.0.0.1:6379?db=3
+```
```
Comment (jenlij) [inline Test-network.md]: add instructions for db-start flag

**Comment by JessicaGreben** [comment]
these docs are good, but I'm sad its this complicated with so many different databases. Oh well, we can hopefully reduce complexity some day in the future.

File: Test-network.md [hunk 3]
```diff
@@ -140,23 +140,59 @@ $ storj-sim network run
...
-However, on setup, you may pass in a redis address for a `redis-server` you're already running locally like in this example: `storj-sim network setup --redis redis://127.0.0.1:6379` 
+```
+// satellite 1
+--live-accounting.db: redis://127.0.0.1:6379?db=2
+--server.revocation-dburl: redis://127.0.0.1:6379?db=3
+```
 
-Note: 
-To set up a redis server locally, you can download redis here: https://redis.io/download. Once downloaded, simply run `redis-server` or `redis-server -port <port>` in a separate terminal window.
+If you're running your own redis server, you can designate the database value to start with flag `--db-start` which defaults to `0`. For example, `storj-sim network --satellites 2 --redis 127.0.0.1:6379 --db-start 3 setup` will start incrementing the dbs from 3 (db=3,4,5,6 for 2 satellites). It assumes all databases from the start value onward are available.
```
Comment (JessicaGreben) [inline Test-network.md]: the only thing that would be nice to change (i think egon commented on this in your PR) is to name `--db-start` something like `--redisdb-start` so its clear its only for redis and not all dbs

File: Test-network.md [hunk 4]
```diff
@@ -140,23 +140,59 @@ $ storj-sim network run
 ### Using Redis
 A few different processes can use redis as their database. Live accounting cache and revocation db both need to use redis on the Satellite. They will exist as separate databases on the same redis instance. For example, live accouning may use `redis://127.0.0.1:6379?db=0` while revocation may use `redis://127.0.0.1:6379?db=1`. 
 
-The default on storj-sim is to create a test redis server automatically and assign databases to processes incrementally based on how many Satellites are run. For example, if you're running 2 satellites, a configuration like the following may be generated.
+By default, storj-sim will start a redis server for each Satellite that is run and write the respective connection string for each database to a redis.conf file. You must have `redis-server` installed locally for this process to run. You can download redis at https://redis.io/download. (On macOS you can install redis with homebrew `brew install redis`)
```
Comment (ifraixedes) [inline Test-network.md]: We should specify which versions of Redis are supported.

Comment (jenlij) [reply Test-network.md]: I think redis 4.0+ should be sufficient. I was using the latest 5.0 but we use 4.0 in gcp

**Pull request merged**